lib/fungus_toast/games/player.ex

```diff
@@ -2,6 +2,19 @@ defmodule FungusToast.Games.Player do
   use Ecto.Schema
   import Ecto.Changeset
 
+  @position_to_attribute_map %{
+    :top_left => :top_left_growth_chance,
```

Comment:
Could we not just do something where we always append `growth_chance`? Then we wouldn't need to maintain a map that seems to do exactly that

Reply:
Perhaps. This way you could change the names up if you wanted, however. *shrug*